testsuite/features/secondary/srv_distro_cobbler.feature

```diff
@@ -6,6 +6,11 @@ Feature: Cobbler and distribution autoinstallation
 
   Background:
     Given I am authorized
+    And I upload ./features/upload_files/install/vmlinuz to /install/Fedora_12_i386/images/pxeboot on server
```

Comment:
`When`

We should hide such technical details from the feature. Please group all these steps into one single step.
Take example on `When(/^I prepare the retail configuration file on server$/) do`
Find a name that suits what all these files are for, and use it to name your new unique step.

Reply:
For me using When must be follow by Then. I really prefer Given / And in background

Reply:
:popcorn: